libs/dh/core/shell/src/lib/dh-core-shell.component.ts

```diff
@@ -14,16 +14,26 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { NgModule } from '@angular/core';
+import { Component, NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
 import { DhLanguagePickerModule } from '@energinet-datahub/dh/globalization/feature-language-picker';
-import { WattModule } from '@energinet-datahub/watt';
+import { WattShellModule } from '@energinet-datahub/watt';
 import { TranslocoModule } from '@ngneat/transloco';
 
-import { ShellComponent } from './shell.component';
+@Component({
+  selector: 'dh-shell',
+  styleUrls: ['./dh-core-shell.component.scss'],
+  templateUrl: './dh-core-shell.component.html',
+})
+export class DhCoreShellComponent {}
 
 @NgModule({
-  declarations: [ShellComponent],
-  imports: [RouterModule, TranslocoModule, WattModule, DhLanguagePickerModule],
+  declarations: [DhCoreShellComponent],
+  imports: [
+    TranslocoModule,
+    DhLanguagePickerModule,
+    RouterModule,
+    WattShellModule,
+  ],
 })
-export class ShellModule {}
+export class DhCoreShellComponentScam {}
```

Comment:
suggestion: rename `DhCoreShellComponentScam` to `DhCoreShellScam`. Do the same for other components introduced by this PR.